StreamingUpdatesRestApi/Program.cs

```diff
@@ -368,5 +424,52 @@ private static IList<SdsSimpleType> GetData()
 
             return data;
         }
+
+        private static IList<WeatherDataType> GetWeatherData()
+        {
+            List<WeatherDataType> data = new ()
+            {
+                new WeatherDataType { Timestamp = DateTime.Now, Humidity = 40.0, Temperature = 25.0 },
+                new WeatherDataType { Timestamp = DateTime.Now, Humidity = 40.1, Temperature = 25.1 },
+            };
+
+            return data;
+        }
+
+        private static bool IsSdsSimpleType(JsonElement update)
+        {
+            string id = update.GetProperty(ResourceId).GetString() ?? string.Empty;
+            return id.StartsWith(SimpleStreamPrefix, StringComparison.OrdinalIgnoreCase);
+        }
+
+        private static void ProcessSimpleStreamUpdate(JsonElement update)
+        {
+            string id = update.GetProperty(ResourceId).GetString();
+            string operation = update.GetProperty(Operation).GetString();
+            ArrayEnumerator events = update.GetProperty(Events).EnumerateArray();
+
+            Console.WriteLine($"id: {id}");
+            Console.WriteLine($"operation: {operation}");
+            foreach (JsonElement jsonDataEvent in events)
+            {
+                SdsSimpleType dataEvent = JsonSerializer.Deserialize<SdsSimpleType>(jsonDataEvent);
```

Comment:
A while back someone mentioned that the structure of the returned event would depend on the type of operation. If that is the case, these process functions might need to account for the different operation types.

Reply:
I expanded the example to include Update, Replace, Remove and RemoveWindow operations. The CLR types (`SdsSimpleType`, `WeatherDataType`), can be used to deserialize updates from Remove and RemoveWindow events. I added a comment on lines 352-3.